tests/testthat/test-tab_afun_cfun.R

```diff
@@ -362,3 +362,25 @@ test_that(".alt_df_row appears in cfun but not in afun.", {
   )
   expect_silent(lyt %>% build_table(ex_adsl, alt_counts_df = alt_tmp))
 })
+
+
+test_that("full alt_counts_df is accessible from afun/cfun via .alt_df_full", {
```

Comment:
hi @gmbecker , i was wondering if you could add explicit test case for using alt_df_full as an argument. thank you